feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -775,14 +796,15 @@ class BillingWrapper(
         }
     }
 
-    private fun createPurchaseParams(
+    private fun buildPurchaseParams(
         storeProduct: GoogleStoreProduct,
-        purchaseOption: PurchaseOption,
+        purchaseOption: GooglePurchaseOption,
         replaceSkuInfo: ReplaceSkuInfo?,
-        appUserID: String
+        appUserID: String,
+        callback: ()
```

Comment:
I don't think this line was meant to be committed? The callback is missing a return type and it is actually not used and causing other compile errors

Reply:
@beylmk I changed how we do this in https://github.com/RevenueCat/purchases-android/pull/676. Lmk if you like that and I can merge that PR into this branch.

Reply:
sorry i think there have been so many merges back into this branch. i'll be more careful to re-check all the changes in the PR before re-requesting reviews. @tonidero thanks, checking it out!

Reply:
@tonidero @vegaro i'd really like to get this merged, it's been open forever and i keep having to rebase things. are we okay leaving it as-is (after i fix this callback error and toni's api tester comment) and toni's work can clean it up? just to avoid more merging errors and people having to open PRs into this?

i can open a separate PR for the callback/Result plan too.

Reply:
Yeah I think it's ok to merge now. About the `Result`, I implemented it on my PR #676 , but lmk if you would prefer something else!